rewrite-maven/src/main/java/org/openrewrite/maven/tree/MavenResolutionResult.java

```diff
@@ -209,6 +209,10 @@ public boolean parentPomIsProjectPom() {
                 .anyMatch(gav -> gav.equals(parentGav));
     }
 
+    public boolean isAggregatingPom() {
+        return !getPom().getSubprojects().isEmpty();
+    }
```

Comment:
"Aggregating pom" is not a term a Maven user would necessarily understand to mean "multi-module"